app/scripts/controllers/attachPVC.js

```diff
@@ -67,6 +67,10 @@ angular.module('openshiftConsole')
       subpage: 'Add Storage'
     });
 
+    $scope.pvcVersion = APIService.getPreferredVersion('persistentvolumeclaims');
+    $scope.quotaVersion = APIService.getPreferredVersion('resourcequotas');
+    $scope.appliedQuotaVersion = APIService.getPreferredVersion('appliedclusterresourcequotas');
```

Comment:
If we're going to shorten the name, I prefer `clusterQuotaVersion`. That makes it more obvious it's cross namespace.

Reply:
I'm good with `clusterQuotaVersion`